src/SldStyleParser.ts

```diff
@@ -967,10 +987,27 @@ class SldStyleParser implements StyleParser {
     let sldOperator: string = (sldOperators.length > 1 && value === null)
       ? sldOperators[1] : sldOperators[0];
 
-    sldComparisonFilter[sldOperator] = [{
-      'PropertyName': [key],
-      'Literal': [value]
-    }];
+    if (sldOperator === 'PropertyIsNull') {
+      // empty, selfclosing Literals are not valid in a propertyIsNull filter
+      sldComparisonFilter[sldOperator] = [{
+        'PropertyName': [key]
+      }];
+    } else if (sldOperator === 'PropertyIsLike') {
+      sldComparisonFilter[sldOperator] = [{
+        '$': {
+          'wildCard': '*',
+          'singleChar': '.',
+          'escape': '!'
```

Comment:
We should think about making this configurable in the future.

Reply:
yep!